services/emailService.js

```diff
@@ -1,34 +1,29 @@
 'use strict';
 
-var http = require('../common/http.js');
+var http = require('../common/httpClient.js');
 var core = require('../common/core.js');
 var Q = require('q');
 
 module.exports = function () {
-    var getIdpLatestEmail = function (userEmail, subject) {
-        return getLatestEmail(core.getConfig().idpEmailMockService.url, userEmail, subject).then(function (response) {
-            // Workaround for issue in ISG Email Stub where dots are sometimes duplicated (due to that registration link gets broken)
-            // Bus to provide more details
-            return JSON.parse(response.body.replace(/\.\./g, '.'));
-        });
-    };
-
     var getIdamLatestEmail = function (userEmail, subject) {
```

Comment:
As now we have only one stub email service handling both type of emails (form idp and our events) I have moved everything to the same method, so basically the replace to resolve the double dot and new lines are done in the same method.